monitor/census.go

```diff
@@ -185,12 +192,13 @@ type (
 	}
 
 	segmentsAverager struct {
-		segments  []segmentCount
-		start     int
-		end       int
-		removed   bool
-		removedAt time.Time
-		tries     map[uint64]tryData // seqNo:try
+		manifestID string
```

Comment:
What is `segmentsAverager` used for? Does adding manifestID to it affect the dimensionality, want to confirm if it's now creating an `averager` per stream as opposed to before per lines `1451`, `1468` in `census.go`, will this affect the functionality or how it's used?

Reply:
`averager` is created per-stream, before and after the change

Reply:
So just including the ID which wasn't present before, got it